iina/MPVController.swift

```diff
@@ -218,10 +218,16 @@ class MPVController: NSObject {
     let scriptPath = Bundle.main.path(forResource: "autoload", ofType: "lua", inDirectory: "scripts")!
     chkErr(mpv_set_option_string(mpv, MPVOption.ProgramBehavior.script, scriptPath))
     
-    let inputConfPath = Bundle.main.path(forResource: "input", ofType: "conf", inDirectory: "config")!
-    chkErr(mpv_set_option_string(mpv, MPVOption.Input.inputConf, inputConfPath))
-    
-    // Receive log messages at warn level.
+    //load keybinding
+    let userConfigs = UserDefaults.standard.dictionary(forKey: PK.inputConfigs)
+    var inputConfPath =  PrefKeyBindingViewController.defaultConfigs["MPV Default"]
+    if let confFromUd = UserDefaults.standard.string(forKey: PK.currentInputConfigName) {
+      let currentConfigFilePath = getFilePath(Configs: userConfigs, forConfig: confFromUd, showAlert: false)
+      if confFromUd != "MPV Default"  &&  currentConfigFilePath != nil {
+        inputConfPath = currentConfigFilePath!
+      }
+    }
```

Comment:
Why check `confFromUd != "MPV Default"` here? I think the following logic is enough:

```swift
if let confFromUd = UserDefaults.standard.string(forKey: PK.currentInputConfigName) {
  if let currentConfigFilePath = getFilePath(Configs: userConfigs, forConfig: confFromUd, showAlert: false) {
    inputConfPath = currentConfigFilePath
  }
}
```